src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo';
+import { getParent } from '../../services/getParent';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
```

Comment:
You can get rid of code duplication.


use

export enum Table {
  Name = 'Name',
  Sex = 'Sex',
  Born = 'Born',
  Died = 'Died',
  Mother = 'Mother',
  Father = 'Father'
}


and make a map using object.values

Reply:
> You can get rid of code duplication.
> 
> use
> 
> export enum Table { Name = 'Name', Sex = 'Sex', Born = 'Born', Died = 'Died', Mother = 'Mother', Father = 'Father' }
> 
> and make a map using object.values

Good point, or just create an array of column names instead of enum